styles/variables-base.less

```diff
@@ -296,108 +305,107 @@
 @agate-progressbar-vertical-width: @agate-progressbar-height;
-@agate-progressbar-small-height: inherit;
+@agate-progressbar-small-height: (@agate-progressbar-height / 2);
 @agate-progressbar-vertical-small-width: @agate-progressbar-small-height;
 @agate-progressbar-border-radius: @agate-slider-border-radius;
 @agate-progressbar-track-transform: @agate-slider-track-transform;
-@agate-progressbar-border-radius: @agate-slider-border-radius;
-@agate-progressbar-vertical-track-transform: inherit;
+@agate-progressbar-vertical-track-transform: none;
 
 // Slider
 // ---------------------------------------
-@agate-slider-knob-active-transform: inherit;
+@agate-slider-knob-active-transform: @agate-translate-center scale(1);
 @agate-slider-knob-border-radius: @agate-slider-knob-height;
-@agate-slider-knob-border-width: inherit;
-@agate-slider-knob-resting-state-scale: ;
+@agate-slider-knob-border-width: 0;
+@agate-slider-knob-resting-state-scale: 1;
 @agate-slider-knob-transform: @agate-translate-center;
-@agate-slider-knob-height: inherit;
+@agate-slider-knob-height: (@agate-slider-height - 6px);
 @agate-slider-knob-width: @agate-slider-knob-height;
 @agate-slider-border-radius: @agate-slider-height;
-@agate-slider-horizontal-margin: inherit;
-@agate-slider-horizontal-padding: inherit;
-@agate-slider-vertical-margin: inherit;
-@agate-slider-vertical-padding: inherit;
+@agate-slider-horizontal-margin: 0 @agate-component-spacing;
+@agate-slider-horizontal-padding: ((@agate-slider-height - @agate-slider-track-height) / 2) (@agate-slider-height / 2);
+@agate-slider-vertical-margin: 0 @agate-component-spacing;
+@agate-slider-vertical-padding: (@agate-slider-height / 2) ((@agate-slider-height - @agate-slider-track-height) / 2);
 @agate-slider-height: @agate-button-height;
 @agate-slider-width: @agate-slider-height;
-@agate-slider-track-height: inherit;
+@agate-slider-track-height: 12px;
 @agate-slider-track-width: @agate-slider-track-height;
-@agate-translate-center: inherit;
-@agate-slider-bg-transform: inherit;
-@agate-slider-track-transform: inherit;
+@agate-translate-center: translate3d(-50%, -50%, 0);
+@agate-slider-bg-transform: none;
+@agate-slider-track-transform: none;
 
 // IncrementSlider
 // ---------------------------------------
-@agate-incrementslider-border-radius: inherit;
+@agate-incrementslider-border-radius: 0;
 @agate-incrementslider-horizontal-margin: @agate-slider-horizontal-margin;
-@agate-incrementslider-horizontal-slider-margin: inherit;
-@agate-incrementslider-horizontal-slider-padding: inherit;
-@agate-incrementslider-horizontal-button-margin: inherit;
-@agate-incrementslider-horizontal-button-decrement-border-radius: inherit;
-@agate-incrementslider-horizontal-button-decrement-border-width: inherit;
-@agate-incrementslider-horizontal-button-increment-border-radius: inherit;
-@agate-incrementslider-horizontal-button-increment-border-width: inherit;
+@agate-incrementslider-horizontal-slider-margin: 0 -1px;
```

Comment:
This `-1px` seems skin-specific. `0` should be sufficient for the base value, and skins that really need a negative 1 horizontal margin are welcome to set one, but I get the impression that this is a pretty specific visual.